app/livechat/server/api/v1/room.js

```diff
@@ -42,6 +43,52 @@ API.v1.addRoute('livechat/room', {
 	},
 });
 
+API.v1.addRoute('livechat/rooms', { authRequired: true }, {
```

Comment:
We use `imports/server/rest` folder to implement the Livechat oauth endpoints, the `server/api/v1` folder is used to store the client-side endpoints, did you get it?
Lets move this implementation to the `imports/server/rest`, ok?